esp.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#check if docker is installed
+if ! /usr/local/bin/docker --version  &> /dev/null; then
+	echo "Docker is not installed. Please install Docker and try again."
+	exit 1
+fi
+
+#Start docker desktop
+open --background -a Docker
+
+#wait for docker to be ready
+while ! docker  info > /dev/null 2>&1; do
+	echo "Waiting for Docker to start..."
+	sleep 5
+done
+
+#run command
+docker compose up & 
+
+#open app
+sleep 7
```

Comment:
@Lyskawa-Aonyx - The comment says, 'I have also fixed the sleep timer to check if the "docker-compose up" command is finished before opening the user's browser.' However, I still see a sleep 7 in the code. Could you please verify if your latest changes have been pushed? If not, could you include this update as part of the stop script issue?